recipes/gegles-spdlog_setup/all/conanfile.py

```diff
@@ -40,7 +40,7 @@ def layout(self):
     def requirements(self):
         self.requires("cpptoml/0.1.1")
         self.requires("spdlog/[>=1.15 <2]")
-        self.requires("fmt/[>=10 <12]")
+        self.requires("fmt/[>=10 <=12]")
```

Comment:
This is likely to cause the same issue in the future: a yet-to-be-released version of spdlog may resolve to a version of fmt that is _newer_ than 12 (e.g. 13), and we'll be back in the same place.

```suggestion
        self.requires("fmt/[*]")
```

Setting the range to `[*]`, the way Conan solves the graph, means that it will use the exact same version of `fmt` that `spdlog` resolved to - perhaps that's a safer way.